### ISSUE TITLE
OpenApi 3.0: format not show in query parameters

### ISSUE THREAD

**Issue by pipy007** [swagger-api/swagger-ui#5128]
Hi.

I have issue with displaying format of query parameters. 
In openapi 3.0 the format of parameters is inside schema tag.

openapi 3.0.1 sample of query parameter:
{
  "name": "contractId",
  "in": "query",
  "description": "Query parameter for contractId.",
  "schema": {
    "type": "integer",
    "format": "int64"
  }
}
swagger 2.0 sample of query parameter:
{
  "name": "contractId",
  "in": "query",
  "description": "Query parameter for contractId.",
  "required": false,
  "type": "integer",
  "format": "int64"
},
display of query parameter:
![image](https://user-images.githubusercontent.com/46778405/51309046-d4fd1e80-1a43-11e9-8e79-a778e3968664.png)
expected behavior:
![image](https://user-images.githubusercontent.com/46778405/51309082-e9d9b200-1a43-11e9-8218-578a4c553eef.png)

### Q&A (please complete the following information)
 - OS: windows
 - Browser: chrome
 - Version: 70
 - Method of installation: dist
 - Swagger-UI version: 3.20.5
 - OpenAPI version: OpenAPI 3.0.1

**Comment by Gozke** [comment]
Hi,
We've noticed that the format is not the only thing that is not displayed. For instance
* minimum
* maxumum 
* pattern

Are also not displayed. 

It would be nice to see the schema related stuff rendered for query parameters the same way they are rendered for model in case of a request body.

**Comment by domdomegg** [comment]
Here's another example. Annoying as swagger-ui works fine with Swagger 2 instead of OpenAPI 3.

## OpenAPI

```yaml
openapi: 3.0.0
info:
  title: Request parameter examples
  version: 1.0.0
paths: 
  /accounts/{accountUuid}:
    get:
      parameters: 
      - name: accountUuid
        in: path
        required: true
        schema:
          type: string
          format: uuid
      - name: createdAfter
        in: query
        required: true
        schema:
          type: string
          format: date
      responses:
        '200':
          description: The account
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/account'
components:
  schemas:
    account:
      type: object
      properties:
        accountUuid:
          type: string
          format: uuid
        creationDate:
          type: string
          format: date
        username:
          type: string
          example: "domdomegg"
```

## Swagger

```yaml
swagger: '2.0'
info:
  title: Request parameter examples
  version: 1.0.0
paths: 
  /accounts/{accountUuid}:
    get:
      parameters: 
      - name: accountUuid
        in: path
        required: true
        type: string
        format: uuid
      - name: createdAfter
        in: query
        required: true
        type: string
        format: date
      responses:
        200:
          description: The account
          schema:
            $ref: '#/definitions/account'
definitions:
  account:
    type: object
    properties:
      accountUuid:
        type: string
        format: uuid
      creationDate:
        type: string
        format: date
      username:
        type: string
        example: "domdomegg"
```

**Comment by webron** [comment]
PRs are welcome.

**Comment by webron** [comment]
PRs are welcome.

**Comment by domdomegg** [comment]
PR that should fix this: #5796 🎉

**Comment by danosborne** [comment]
Can we get some momentum behind PR #5796? Looks like it does the job for me. Would be great to see this in a release soon.
